Automattic/jetpack
# Readymade Templates: Generate content task definition

Related to:
- https://github.com/Automattic/dotcom-forge/issues/7500

## Proposed changes:

* Added a new task definition for generating content with AI on the RT flow.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Apply this code to your sandbox
* Sandbox your API
* Go to `/patterns` and on Chrome filter network requests by `launchpad`
* The API should include the new task.

<img width="865" alt="Screenshot 2024-07-24 at 17 37 53" src="https://github.com/user-attachments/assets/f7bf3655-852b-4f3f-a006-af47fcc33a06">
